devfile.yaml

```diff
@@ -45,25 +45,10 @@ commands:
         type: exec
         command: |
           set -o xtrace;
-          rm -rf build;
```

Comment:
`workdir` should be `${CHE_PROJECTS_ROOT}/che-plugin-registry`

Reply:
fixed